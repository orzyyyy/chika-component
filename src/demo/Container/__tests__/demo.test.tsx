import React from 'react';
import { shallow } from 'enzyme';
import 'nino-cli/scripts/setup';
import ContainerDemo from '..';

describe('ContainerDemo', () => {
  it('render correctly', () => {
    const wrapper = shallow(<ContainerDemo />);
    expect(wrapper).toMatchSnapshot();
  });
});
