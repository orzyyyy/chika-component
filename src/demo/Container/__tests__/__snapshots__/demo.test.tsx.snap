// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContainerDemo render correctly 1`] = `
<div
  className="container-demo"
>
  <DataController
    domain="."
    menuId={1392}
    tableId={10874}
  >
    <ul>
      <li>
        <div
          className="left"
        >
          <label>
            名称：
          </label>
          <label
            data-key="pjnm"
          />
        </div>
        <div
          className="right"
        >
          <label>
            坝高：
          </label>
          <label
            data-key="dam_width"
          />
        </div>
      </li>
      <li>
        <div
          className="left"
        >
          <label>
            坝长：
          </label>
          <label
            data-key="crest_length"
          />
        </div>
        <div
          className="right"
        >
          <label>
            主坝类型：
          </label>
          <label
            data-key="retain_dam_type"
          />
        </div>
      </li>
    </ul>
  </DataController>
</div>
`;
